src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-document-card/sw-order-document-card.spec.js

```diff
@@ -248,6 +248,34 @@ describe('src/module/sw-order/component/sw-order-document-card', () => {
         setActivePinia(createPinia());
     });
 
+    it('Should test filename extraction from content-disposition header', async () => {
+        wrapper = await createWrapper();
+
+        let response = {
+            headers: {
+                'content-disposition': 'attachment; filename="normal-filename.pdf"',
```

Comment:
this case would have a space in the real world right? otherwise the quotes are not needed / added
```suggestion
                'content-disposition': 'attachment; filename="normal filename.pdf"',
```